portal/src/graphql/adminapi/ChangePasswordScreen.tsx

```diff
@@ -166,8 +168,9 @@ const ChangePasswordScreen: React.VFC = function ChangePasswordScreen() {
         state.sendPassword,
         state.setPasswordExpired
       );
+      await refetchUser();
```

Comment:
Do not do this. Instead, you should modify resetPasswordMutation.graphql to fetch the updated fields.